uclapi.openapi.json

```diff
@@ -0,0 +1,4256 @@
+            "description": "The total number of used seats at time t."
+          },
+          "sensors_occupied" : {
+            "type" : "number",
+            "example" : 3,
+            "description": "Number of taken / occupied seats in that section of the library."
+          },
+          "name" : {
+            "type" : "string",
+            "example" : "Open Area",
+            "description": "The name of the survey (library)."
+          },
+          "id" : {
+            "type" : "string",
+            "example" : "72",
+            "description": "ID of the library region (map)."
+          }
+        }
+      },
+      "survey_with_maps" : {
+        "properties" : {
+          "name" : {
+            "type" : "string",
+            "example" : "Teaching Rooms",
+            "description": "Name of the library region (map)."
+          },
+          "id" : {
+            "type" : "string",
+            "example" : "73",
+            "description": "ID of the library region (map)."
+          },
+          "sensors_absent" : {
+            "type" : "number",
+            "example" : 72,
+            "description": "Number of free seats in that section of the library."
+          },
+          "sensors_total" : {
+            "type" : "number",
+            "example" : 80,
+            "description": "The total number of used seats at time t."
+          },
+          "sensors_occupied" : {
+            "type" : "number",
+            "example" : 3,
+            "description": "Number of taken / occupied seats in that section of the library."
+          },
+          "maps" : {
+            "type" : "array",
+            "items" : {
+              "$ref" : "#/components/schemas/average_with_name_and_id"
+            }
+          }
+        }
+      },
+      "svg" : {
+        "type" : "object",
+        "properties" : {
+          "xmlns:ev" : {
+            "type" : "string",
+            "example" : "http://www.w3.org/2001/xml-events",
+            "xml" : {
+              "attribute" : true
+            }
+          },
+          "xmlns:xlink" : {
+            "type" : "string",
+            "example" : "http://www.w3.org/1999/xlink",
+            "xml" : {
+              "attribute" : true
+            }
+          },
+          "xmlns" : {
+            "type" : "string",
+            "example" : "http://www.w3.org/2000/svg",
+            "xml" : {
+              "attribute" : true
+            }
+          },
+          "g" : {
+            "type" : "object",
+            "properties" : {
+              "transform" : {
+                "type" : "string",
+                "example" : "scale(0.02, 0.02)",
+                "xml" : {
+                  "attribute" : true
+                }
+              },
+              "image" : {
+                "type" : "object",
+                "properties" : {
+                  "width" : {
+                    "type" : "string",
+                    "example" : "28329.0",
```

Comment:
nit: same as above string for number value

Reply:
I think this is a string because it's a HTML SVG so all the attributes would be strings